tests/drivers/uart/uart_async_api/src/test_uart.h

```diff
@@ -19,7 +19,9 @@
 
 /* RX and TX pins have to be connected together*/
 
-#if DT_NODE_EXISTS(DT_NODELABEL(dut))
+#if DT_NODE_EXISTS(DT_ALIAS(uartasync))
+#define UART_NODE DT_ALIAS(uartasync)
```

Comment:
This is not right as it will cause confusion when looking at the SoC/board DTS file. A simple `uartasync` would give an impression that it is the only async uart supported, which is usually not true. You can do an alias in the overlay file (above) and designate the correct UART as `dut`.

Reply:
Remove?